app/views/partials/_pet_card.html.erb

```diff
@@ -1,26 +1,34 @@
 <div class="col-lg-4 col-md-6 p-0">
   <%= link_to adoptable_pet_path(pet), class: 'text-decoration-none' do %>
     <div class="card card-hover m-4" style='overflow: hidden'>
```

Comment:
I moved the 'Adoption Pending' badge to the top right of the image so there was more room for the like functionality. I also prefer this placement of this badge to before. Thoughts?

I also make the image and the pet name clickable to go to the pet show page, instead of the whole card. This allows the user to click the like/unlike independently in the pets index view, and is consistent with how other websites link their cards. Usually details are not clickable, only names and images.

Reply:
Looks and sounds good to me! I also made the fosters cards with only the image clickable so this is consistent there.